src/Uno.UWP/ApplicationModel/DataTransfer/StandardDataFormats.Android.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Windows.ApplicationModel.DataTransfer
+{
+	public partial class StandardDataFormats
+	{
+		public static string Html { get => "HTML Format"; }
+		public static string Text { get => "Text"; }
+		public static string Uri { get => "UniformResourceLocatorW"; }

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this property type to 'System.Uri'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5540286)